common/ledger/blkstorage/fsblkstorage/blockfile_archiver.go

```diff
@@ -205,20 +178,12 @@ func (arch *blockfileArchiver) deleteArchivedBlockfile(fileNum int) error {
 }
 
 // isNeedArchiving - returns whether archiving should be triggered or not
-func isNeedArchiving(blockfileFolder string, keepFileNum int) bool {
-	loggerArchive.Debugf("blockfileFolder=%s, keepFileNum=%d", blockfileFolder, keepFileNum)
-
-	files, err := ioutil.ReadDir(blockfileFolder)
-	if err != nil {
-		loggerArchive.Error(err)
-		return false
-	}
-
-	if len(files) > keepFileNum {
-		loggerArchive.Debugf("%d blockfile(s) should be archived", len(files)-keepFileNum)
-		return true
-	}
+func (arch *blockfileArchiver) isNeedArchiving() bool {
 
-	loggerArchive.Debug("There is no blockfile to be archived yet")
+	remoteArchivedBlockHeight := blockarchive.GossipService.ReadArchivedBlockHeight(common.ChannelID(arch.chainID))
+	localCurrentBlockHeigh := arch.mgr.getBlockchainInfo().Height
+	localDiscardedBlockHeight := arch.currDiscardedBlockHeigh
+	// numKeepLatestBlocks := blockarchive.NumKeepLatestBlocks
+	loggerArchive.Infof("local : %d  vs  archived : %d  vs  discarded : %d", localCurrentBlockHeigh, remoteArchivedBlockHeight, localDiscardedBlockHeight)
 	return false
```

Comment:
This function is always returning "false".
What is the purpose for this function?

Reply:
Sorry. This function has not been used any more. I delete it.